src/pages-and-resources/discussions/app-config-form/apps/shared/AppExternalLinks.jsx

```diff
@@ -13,26 +15,30 @@ function AppExternalLinks({
   title,
 }) {
   const { contactEmail, ...links } = externalLinks;
-  const linkTypes = Object.keys(links);
+  const linkTypes = Object.keys(links).filter(key => links[key]);
   return (
     <div className="pt-4">
-      <h4>{intl.formatMessage(messages.linkTextHeading)}</h4>
-      <div className="small text-muted">
-        {linkTypes.map((type) => (
-          links[type] && (
-            <div key={type}>
-              <Hyperlink
-                destination={externalLinks[type]}
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                {intl.formatMessage(messages[type], { title })}
-              </Hyperlink>
-            </div>
-          )
-        ))}
-        <hr />
-        {contactEmail && (
+      {linkTypes.length
```

Comment:
maybe `linkTypes.length &&`?

Reply:
It starts showing 0 on UI with && operator if there is no element present, hence explicitly passing null in that case.